composer.json

```diff
@@ -178,5 +178,11 @@
                 "extension.neon"
             ]
         }
+    },
+    "scripts": {
+        "analyse:phpstan": "phpstan analyse"
+    },
+    "scripts-descriptions": {
+        "analyse:phpstan": "Run static analysis"
     }
```

Comment:
Sweet, so we can run this for the entire monorepo, all at once?

Reply:
Yes, everything is currently setup for it to run on all the monorepo at once, including the workflow.